purchase_batch_invoicing/wizards/purchase_batch_invoicing.py

```diff
@@ -60,6 +61,9 @@ def _purchase_order_domain(self, ids=None):
             domain[1] = ("id", "in", pos.ids)
         return domain
 
+    def _prepare_batch_invoice_vals(self, partner):
```

Comment:
You can add a docstring documenting the purpose of it:

"""Allow to override the invoice defaults by a third module. i.e.: set invoice type to in_refund"""